graphics/grafx2/Portfile

```diff
@@ -4,6 +4,8 @@ PortSystem          1.0
 
 name                grafx2
 version             2.6
+revision            1
+set commit          75579a9456115f8475b0a0c8ecf23fe17599eeae
```

Comment:
Did you ask upstream to tag a new release? Keeping version at 2.6 but setting another commit is discouraged, because it could cause confusion. If upstream couldn't tag a new release in time and the bugfixes are important, please add a comment stating that this commit is not tag v2.6.

Reply:
upstream traditionally do not tag "bug fixes" releases. It uses a "commit count" (legacy of the time they used SVN)
commit  75579a94  is v2.5.2490
Should I add a comment in the portfile to state that ?

Reply:
Yes, please add a comment explaining the situation, then this PR can be merged.